### PR TITLE
Incrementalified

### PR DESCRIPTION
**Pull request by jglick**

### CODE REVIEW

File: src/main/java/org/jenkinsci/plugins/durabletask/executors/ContinuedTask.java [hunk 1]
```diff
@@ -49,7 +49,7 @@
      */
     boolean isContinued();
 
-    @Restricted(DoNotUse.class) // implementation
+    @Restricted(NoExternalUse.class) // implementation
```
Comment (jglick) [inline src/main/java/org/jenkinsci/plugins/durabletask/executors/ContinuedTask.java]: https://github.com/kohsuke/access-modifier/pull/12

File: pom.xml [hunk 2]
```diff
@@ -42,13 +44,13 @@
...
         <url>https://github.com/jenkinsci/${project.artifactId}-plugin</url>
-        <tag>HEAD</tag>
+        <tag>${scmTag}</tag>
     </scm>
     <dependencies>
         <dependency>
             <groupId>org.jenkins-ci.test</groupId>
             <artifactId>docker-fixtures</artifactId>
-            <version>1.6</version>
+            <version>1.8</version>
```
Comment (oleg-nenashev) [inline pom.xml]: JTH still does not bundle it iiuc

File: src/main/java/org/jenkinsci/plugins/durabletask/executors/ContinuedTask.java (hunk 1 above)
Comment (jglick) [reply src/main/java/org/jenkinsci/plugins/durabletask/executors/ContinuedTask.java]: or perhaps not

File: pom.xml (hunk 2 above)
Comment (jglick) [reply pom.xml]: No, it is a separate dependency.

File: src/test/resources/org/jenkinsci/plugins/durabletask/CentOSFixture/Dockerfile [hunk 3]
```diff
@@ -1,7 +1,7 @@
 FROM centos:7.4.1708
 RUN yum -y install \
-        openssh-server-7.4p1-13.el7_4 \
-        java-1.8.0-openjdk-headless-1.8.0.151-1.b12.el7_4 \
+        openssh-server \
```
Comment (svanoort) [inline src/test/resources/org/jenkinsci/plugins/durabletask/CentOSFixture/Dockerfile]: @jglick Why?

Comment (jglick) [reply src/test/resources/org/jenkinsci/plugins/durabletask/CentOSFixture/Dockerfile]: See https://github.com/jenkinsci/durable-task-plugin/pull/72/commits/621cfbef0232e176059e834fa465d8a3fcff8272 and a long series of PRs in `docker-fixtures`.

**Pull request merged**